engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -19,34 +19,40 @@
 import org.terasology.rendering.opengl.BaseFBOsManager;
 import org.terasology.rendering.opengl.FBOManagerSubscriber;
 import com.google.common.base.Objects;
-import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
 
 import static org.lwjgl.opengl.EXTFramebufferObject.GL_FRAMEBUFFER_EXT;
 import static org.lwjgl.opengl.EXTFramebufferObject.glBindFramebufferEXT;
 
 /**
- * TODO: Add javadocs
+ * Binds the given FBO to the GL_FRAMEBUFFER target, making it the read framebuffer target as well as
+ * the draw framebuffer target.
+ *
+ * This is useful for operations such as off-screen rendering.
```

Comment:
Sounds reasonably, should be as easy as changing a single flag.. I'll take care of it in a separate commit.